cmd/kubectl-testkube/commands/scripts/watch_script_execution.go

```diff
@@ -20,8 +21,18 @@ func NewWatchExecutionCmd() *cobra.Command {
 			}
 
 			client, _ := GetClient(cmd)
+			execution, err := client.GetExecution("", executionID)
+			if err != nil {
 
-			watchLogs(executionID, client)
+				ui.Failf("execution result retrievel failed with err %s", err)
+			}
+
+			if execution.ExecutionResult.Status == testkube.ExecutionStatusError ||
```

Comment:
Look at methods in execution. There is that logic implemented already

Reply:
Thanks, fixed.